egs2/TEMPLATE/tts1/tts.sh

```diff
@@ -287,28 +293,29 @@ if [ ${stage} -le 5 ] && [ ${stop_stage} -ge 5 ]; then
 
     for dset in "${dev_set}" ${eval_sets}; do
         _data="${data_feats}/${dset}"
-        _dir="${tts_exp}/decode_${dset}${decode_tag}"
-        _logdir="${_dir}/logdir"
+        _dir="${tts_exp}/${decode_tag}_${dset}"
+        _logdir="${_dir}/log"
         mkdir -p "${_logdir}"
 
         # 0. Copy feats_type
-        cp ${_data}/feats_type ${_dir}/feats_type
+        cp "${_data}/feats_type" "${_dir}/feats_type"
         # 1. Split the key file
         key_file=${_data}/token_int
         split_scps=""
-        _nj=$((${decode_nj}<$(<${key_file} wc -l)?${decode_nj}:$(<${key_file} wc -l)))
+        _nj=$((decode_nj<$(<${key_file} wc -l)?decode_nj:$(<${key_file} wc -l)))
         for n in $(seq ${_nj}); do
             split_scps+=" ${_logdir}/keys.${n}.scp"
         done
+        # shellcheck disable=SC2086
         utils/split_scp.pl "${key_file}" ${split_scps}
 
         # 2. Submit decoding jobs
         log "Decoding started... log: '${_logdir}/tts_decode.*.log'"
+        # shellcheck disable=SC2086
         ${_cmd} --gpu "${_ngpu}" JOB=1:"${_nj}" "${_logdir}"/tts_decode.JOB.log \
             python3 -m espnet2.bin.tts_decode \
                 --ngpu "${_ngpu}" \
-                --data_path_and_name_and_type "${_data}/token_int,text,text_int" \
-                --key_file "${_logdir}"/keys.JOB.scp \
+                --data_path_and_name_and_type "${_logdir}/keys.JOB.scp,text,text_int" \
```

Comment:
@kamo-naoyuki How do you think this change? For me, this is more simple.

Reply:
Thank you for reading deeply.
I cared about the second inputs case for this.
If directly inputting from `data_path_and_name_and_type`, then the option of `key_file` is hidden and then users may think that the second scp file also needs to be split.
(and I think you'll add such option to `tts.sh`, if not so, your idea is better.)

Reply:
I see. `--key_file` is nice when we also want to use such as `--data_path_and_name_and_type x_vector.scp,spembs,...` because we do not need to split scp multiple times. I will leave comments about it.